game/db.py

```diff
@@ -938,11 +938,12 @@ def unit_type_name_2(unit_type) -> str:
 
 def unit_get_expanded_info(country_name: str, unit_type, request_type: str) -> str:
     original_name = unit_type.name and unit_type.name or unit_type.id
+    id = unit_type.id
```

Comment:
```suggestion
```

Avoid naming anything `id` in Python; that's a builtin function that gets shadowed if you do. Methods and members are okay since they can't be referred to without a prefix, but globals, locals, functions, classes, etc shouldn't use that name.

Reply:
Good to know :)